client/src/helpers/api.js

```diff
@@ -201,6 +201,26 @@ export const updateUserProfile = async (name, email) => {
   });
 };
 
+export const isAdminLoggedIn = async () => {
+  const token = localStorage.getItem("jwtToken");
+  const currentUserId = await instance
+    .get("/user/profile", {
+      headers: {
+        Authorization: `Bearer ${token}`
+      }
+    })
+    .then(res => res.data.user_id);
+
+  const result = await instance.get(`/api/user-roles/${currentUserId}`, {
```

Comment:
this `instance` is changed now. Change it to createInstane(). Have a look at other APIs in this file